tools/cldr-unittest/src/org/unicode/cldr/unittest/TestInheritance.java

```diff
@@ -247,18 +247,18 @@ public void TestLikelyAndDefaultConsistency() {
         // find multiscript locales
         Relation<String, String> base2scripts = Relation.of(
             new TreeMap<String, Set<String>>(), TreeSet.class);
-        Map<String, String> parent2default = new TreeMap<String, String>();
-        Map<String, String> default2parent = new TreeMap<String, String>();
+        Map<String, String> parent2default = new TreeMap<>();
+        Map<String, String> default2parent = new TreeMap<>();
         Relation<String, String> base2locales = Relation.of(
             new TreeMap<String, Set<String>>(), TreeSet.class);
 
-        Set<String> knownMultiScriptLanguages = new HashSet<String>(Arrays.asList("bm", "ha"));
+        Set<String> knownMultiScriptLanguages = new HashSet<>(Arrays.asList("bm", "ha"));
         // get multiscript locales
         for (String localeID : availableLocales) {
             String script = ltp.set(localeID).getScript();
             final String base = ltp.getLanguage();
-            if (!availableLocales.contains(base)) {
-                errln("Missing base locale for: " + localeID);
+            if (!availableLocales.contains(base) && !base.equals("und")) {
+                errln("Missing lang-subtag for: " + localeID);
```

Comment:
in https://github.com/unicode-org/cldr/pull/757 I made sure to print out the `base` locale also, because that's actually the one that's missing (not `localeID`)